features/step_definitions/background_step.rb

```diff
@@ -0,0 +1,5 @@
+Given("the following products exist:") do |table|
+  table.hashes.each do |attrs|
```

Comment:
`attrs` is okay. `product_attributes` is better. Clarity is good

Reply:
forgot about this one. 1 sec.